specs/AgriFood/AgriCrop/schema.json

```diff
@@ -0,0 +1,110 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/AgriCrop/schema.json",
+    "title": "FIWARE - Agri Food",
+    "description": "This entity contains a harmonised description of a generic crop. This entity is primarily associated with the agricultural vertical and related IoT applications.",
+    "type": "object",
+    "properties": {
+	"id": {
+	    "type": "string",
+	    "format": "uri"
+	},
+	"type": {
+	    "type": "string",
+	    "enum": ["AgriCrop"],
+	    "description": "NGSI Entity Type"
+	},
+	"@context": {
```

Comment:
no need to encode @context in the Schema

Reply:
Fixed in (https://github.com/Ficodes/dataModels/commit/9fc299b36222457f2e78536bc731cb9f0222258a)